development/compiling/compiling_for_windows.rst

```diff
@@ -184,11 +185,7 @@ the ``vsproj=yes`` parameter, like this::
 You will be able to open Godot's source in a Visual Studio solution now,
 and able to build Godot using Visual Studio's **Build** button.
 
-If you need to edit the build commands, they are located in
-"Godot" project settings, NMAKE sheet. SCons is called at the end of
-the commands. If you make a mistake, copy the command from one of the
-other build configurations (debug, release_debug, release) or
-architectures (Win32/x64); they are equivalent.
```

Comment:
I didn't assess how relevant this might still be but it seems to partially overlap with instructions in the dedicated page to configure VS. If some of it is missing there, that's where it should be added.

Reply:
Yeah, I think it's fair to offload this information to the main page about configuring VS.